packages/extension-vscode/src/server.ts

```diff
@@ -3,8 +3,7 @@ import { TextDocument } from 'vscode-languageserver-textdocument';
 
 import { Analyzer } from './utils/analyze';
 import { QuickFixActionProvider } from './quickFixProvider';
-
-import * as notifications from './utils/notifications';
+import { getWehbhintConfigPath, ignoreHintGlobally, ignoreHintPerProject, ignoreProblemInHints } from './webhintUtils';
```

Comment:
@vidorteg I didn't change it here, but as we finish this up, I'd prefer keeping filenames fully lower-case for easier portability cross-platform, so `webhint-utils` in this case (helps avoid issues where casing may be inconsistent in references versus the actual filename).